tcf_website/static/css/site/main.css

```
@@ -23,3 +23,5 @@
 @import url('./components/combo_dropdown.css');
 @import url('./components/misc.css');
 @import url('./components/calendar.css');
+@import url('./components/course_card.css');
```

Stylelint (17.6.0)

[error] Expected "url('./components/course_card.css')" to be "'./components/course_card.css'" (import-notation)

(import-notation)